content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/overview.md

```diff
@@ -58,10 +58,20 @@ Since each entity with a multiplayer component is replicated across the network,
 
 The roles offered for O3DE multiplayer hosts are:
 
-* **Client**: The lowest privilege role for a component. The smallest possible subset of network properties are replicated to this role, and its behavior is strictly read-only. Use this role when working with entities that should contain only presentation logic and act as a proxy for invoking RPCs. Examples of entities which should use this role are AIs and entities under the control of another host.
-* **Autonomous**: A role with the _illusion_ of write access. Autonomous roles are usually assigned to components directly under local user control, and receive a larger amount of network information than a Client role. Autonomous roles can also take advantage of predictive networking.
-* **Authority**: This role has ultimate authority. It has full write access, and sees all network properties on the component.
+* **Client** (`NetEntityRole::Client`): The lowest privilege role for a component. The smallest possible subset of network properties are replicated to this role, and its behavior is strictly read-only. Use this role when working with entities that should contain only presentation logic and act as a proxy for invoking RPCs. Examples of entities that should use this role are AIs and entities under the control of another host.
```

Comment:
This may be inaccurate, please check with the networking team.

Reply:
Generally accurate. The only change I would suggest is that you don't get to chose the role a networked entity is assigned, that's all handled automatically by the multiplayer gem itself. Hence I might change it to be more like this:
`* **Client** (`NetEntityRole::Client`): The lowest privilege role for a component. The smallest possible subset of network properties are replicated to this role, and its behavior is strictly read-only. Use this role when working with entities that should contain only presentation logic and act as a proxy for invoking RPCs. Examples of entities that should use this role are AIs and entities under the control of another host.`

Reply:
If that's the case, I'd go even further to specify that the Multiplayer Gem automatically assigns and handles the roles of networked entities at the top of this section. 

@kberg-amzn You omitted the example sentence in your suggested rewrite. I just wanted to verify whether that example still sounds accurate?